validmind/tests/model_validation/ragas/ContextUtilization.py

```diff
@@ -0,0 +1,125 @@
+# Copyright © 2023-2024 ValidMind Inc. All rights reserved.
+# See the LICENSE file in the root of this repository for details.
+# SPDX-License-Identifier: AGPL-3.0 AND ValidMind Commercial
+
+import warnings
+
+import plotly.express as px
+from datasets import Dataset
+
+from validmind import tags, tasks
+
+from .utils import get_ragas_config, get_renamed_columns
+
+
+@tags("ragas", "llm", "retrieval_performance")
+@tasks("text_qa", "text_generation", "text_summarization", "text_classification")
+def ContextUtilization(
```

Comment:
Can you pass this description through the test description script so its format is consistent with the others?

Reply:
yep will do